cruise-control-metrics-reporter/src/main/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporter.java

```diff
@@ -137,45 +153,107 @@ public void configure(Map<String, ?> configs) {
         LOG.warn("Cruise Control metrics topic auto creation was disabled", e);
       }
     }
-
   }
 
-  private NewTopic createNewTopicFromReporterConfig(CruiseControlMetricsReporterConfig reporterConfig)
-  throws CruiseControlMetricsReporterException {
+  protected NewTopic createNewTopicFromReporterConfig(CruiseControlMetricsReporterConfig reporterConfig)
```

Comment:
Is it possible to make this static and move it to `CruiseControlMetricsUtils` with a JavaDoc?

Reply:
I feel the content of this method is specific to metric topic, I would like to keep it in this class.